core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -114,6 +118,22 @@ object ConsumerGroupCommand extends Logging {
     }
   }
 
+  def printDeletedOffsets(error: Option[Throwable], result: Map[TopicPartition, Throwable]): Unit = {
+    if (error.isDefined)
+      printError(s"Deletion of offsets failed due to: ${error.get.getCause.getMessage}")
```

Comment:
I would hide it as some errors are important for the user (e.g. group not exists).

Reply:
My concern is mainly that we may not get a good cause message for an unexpected error. So if we log the error at debug, then it wouldn't clutter the output by default and users can increase the log level to find the trace.

Reply:
I understand but without this the use would get nothing back from the tool by default in case of error. Let’s say it put an unknown group, he would not get any thing in the output which is weird.

Perhaps we could handle the main expected errors and use an generic error message for the others with details in debug. What do you think?

Reply:
btw, the Throwable here is basically the Error send back by the coordinator so the message will be one of the Errors. Nothing else.

Reply:
hum.. it could be transport errors as well I guess. so what about the following:

if the Throwable is an known Error as defined in Errors, we show it. Otherwise, we display generic error message and put details as debug.

Reply:
I have the change the way I handle top level errors. I did not realise that there is already a top level error handler which is used by all commands. I use it now as it is more consistent with other commands.